src/Sulu/Bundle/AdminBundle/Resources/js/components/MultiAutoComplete/multiAutoComplete.scss

```diff
@@ -29,9 +29,9 @@ $multiAutoCompleteDisabledBackgroundColor: $wildSand;
         display: flex;
         justify-content: center;
         align-items: center;
-        height: calc(100% - 10px);
+        align-self: stretch;
```

Comment:
one of these styling changes increases the height of the autocomplete input field when the loader is displayed for me. can be fixed by setting the size of the loader to `16px` in the following file: https://github.com/sulu/sulu/blob/master/src/Sulu/Bundle/AdminBundle/Resources/js/components/MultiAutoComplete/MultiAutoComplete.js#L186